cmake/CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ cmake_dependent_option(onnxruntime_USE_FULL_PROTOBUF "Link to libprotobuf instea
 option(tensorflow_C_PACKAGE_PATH "Path to tensorflow C package installation dir")
 option(onnxruntime_ENABLE_LANGUAGE_INTEROP_OPS "Enable operator implemented in language other than cpp" OFF)
 option(onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS "Dump debug information about node inputs and outputs when executing the model." OFF)
+option(onnxruntime_ENABLE_SQL "Build dump debug information about node inputs and outputs with support for sql database." OFF)
```

Comment:
>onnxruntime_ENABLE_SQL [](http://example.com/codeflow?start=7&length=22)

can the name be more specific?
e.g., onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS_ENABLE_DUMP_TO_SQLDB
also, if it only makes sense for it to be enabled when onnxruntime_DEBUG_NODE_INPUTS_OUTPUTS is, can use cmake_dependent_option()

Reply:
Yeah, changed as suggested.